addon/mixins/axis-titles.js

```diff
@@ -117,8 +134,13 @@ const AxisTitlesMixin = Ember.Mixin.create({
    * Position of y axis title on the x axis
    * @type {Number}
    */
-  yAxisPositionX: Ember.computed('graphicHeight', 'labelWidthOffset', function(){
-    return -(this.get('graphicHeight') / 2 + this.get('labelWidthOffset'));
+  yAxisPositionX: Ember.computed('graphicHeight', 'labelWidthOffset',
+      'yTitleVerticalOffset', function(){
+    var position = -(this.get('graphicHeight') / 2 + this.get('labelWidthOffset'));
```

Comment:
I'm guessing there's a slight mistake here: `this.get('labelWidthOffset')`, should probably be `this.get('labelHeightOffset')`